pkg/api/platformapi/allocatorapi/vacate.go

```diff
@@ -579,6 +582,47 @@ func CheckVacateFailures(failures *models.MoveClustersDetails, filter []string,
 	return merr.ErrorOrNil()
 }
 
+// GetVacateRequestByClusterID makes models.MoveClusterRequest object which contains a cluster ID
+// and the object will be set to body of an API call which will retrieve calculatedplan data to be used to move a node.
+func GetVacateRequestByClusterID(clusterID, clusterType string) *models.MoveClustersRequest {
```

Comment:
This function should be private since it's only ever called in `newMoveClusterParams` which is a private function, I don't see why we should expand our public API.

Reply:
Oh! I didn't even think about that. Thank you very much for pointing this out.